backend/src/modules/tasks/constants.ts

```diff
@@ -0,0 +1,4 @@
+import { REST_API_PATH } from '../../constants/paths';
+
+export const SHORT_TASK_MODULE_PATH = `/tasks`;
+export const FULL_TASK_MODULE_PATH = `${REST_API_PATH}${SHORT_TASK_MODULE_PATH}`;
```

Comment:
oks :)
Choć tu musimy jakieś fajne nazwy wymyślić :)

Ale na razie musi być, to co mamy :)